examples/view_25d_map.html

```diff
@@ -59,8 +59,8 @@
             // Add a WMS imagery source
             var wmsImagerySource = new itowns.WMSSource({
                 extent: extent,
-                name: 'Ortho2009_vue_ensemble_16cm_CC46',
-                url: 'https://download.data.grandlyon.com/wms/grandlyon',
+                name: 'ortho_2009',
```

Comment:
ortho_latest ou cela pose probleme ?

Reply:
Yikes! I mixed `ortho_2009` and `ortho_latest` layer names...
Since `ortho_2009` seems to be deprecated, we should migrate to `ortho_latest`!